desktop/src/main/document/index.ts

```diff
@@ -1,63 +1,42 @@
 import { ipcMain } from 'electron'
-import { Document, documents } from 'stencila'
+import { documents } from 'stencila'
 import { CHANNEL } from '../../preload'
 
-const documentRefs: Record<string, Document | undefined> = {}
-
-const getDocument = (filePath: string): Document => {
-  let doc = documentRefs[filePath]
-  if (doc) {
-    return doc
-  }
-
-  doc = documents.open(filePath)
-  documentRefs[filePath] = doc
-
-  return doc
-}
-
-const closeDocument = (filePath: string) => {
-  documents.unsubscribe(filePath, ['encoded:html'])
-  documents.close(filePath)
-  delete documentRefs[filePath]
-}
-
 export const registerDocumentHandlers = () => {
   ipcMain.handle(
     CHANNEL.GET_DOCUMENT_CONTENTS,
     async (ipcEvent, filePath: string) => {
-      getDocument(filePath)
-      documents.subscribe(filePath, ['modified'], (_topic, docEvent) => {
+      let documentId = documents.open(filePath).id
```

Comment:
Having two `documents.open(filePath)` in this file doesn't seem right. Instead, I imagine that the `AppDocumentPane` would open the document (when user clicks on a file in the sidebar) and then pass the `documentId` rather than the `filePath` to the channel handlers.